src/plugins/controls/common/control_group/types.ts

```diff
@@ -36,9 +36,15 @@ export interface ControlGroupInput extends EmbeddableInput, ControlInput {
 /**
  * Only parts of the Control Group Input should be persisted
  */
+export const persistableControlGroupInputKeys = [
```

Comment:
Nice usage of `as const` here. I assume this is meant to link the keys used for diffing directly to the `PersistableControlGroupInput` type?

to link it even further, it might be nice to annotate this with `Array<keyof ControlGroupInput>`

Reply:
Good call - done in 48afa7cf56bee5e371e40dc99f32455fb65c2acc 🎉